buildroot-external/rootfs-overlay/etc/dhcp/dhclient-exit-hooks.d/timesyncd

```diff
@@ -0,0 +1,55 @@
+# This file copied from http://deb.debian.org/debian/pool/main/s/systemd/systemd_246.1-1.debian.tar.xz
+# This hook allows for DHCP to define NTP servers for timesyncd
+
+TIMESYNCD_CONF=/run/systemd/timesyncd.conf.d/01-dhclient.conf
+
+timesyncd_servers_setup_remove() {
+        if [ ! -d /run/systemd/system ]; then
+                return
+        fi
+        if [ ! -x /lib/systemd/systemd-timesyncd ]; then
+                return
+        fi
+
+        if [ -e $TIMESYNCD_CONF ]; then
+                rm -f $TIMESYNCD_CONF
+                systemctl try-restart systemd-timesyncd.service || true
+        fi
+}
+
+timesyncd_servers_setup_add() {
+        if [ ! -d /run/systemd/system ]; then
+                return
+        fi
+        if [ ! -x /lib/systemd/systemd-timesyncd ]; then
+                return
+        fi
+
+        if [ -e $TIMESYNCD_CONF ] && [ "$new_ntp_servers" = "$old_ntp_servers" ]; then
+                return
+        fi
+
+        if [ -z "$new_ntp_servers" ]; then
+                timesyncd_servers_setup_remove
+                return
+        fi
+
+        mkdir -p $(dirname $TIMESYNCD_CONF)
+        cat <<EOF > ${TIMESYNCD_CONF}.new
```

Comment:
This will not work because of RO filesystem. You need use tmp/

Reply:
Even `/run` is RO? That is where stuff like PID files and UNIX sockets would go. I believe I was able to write to it when I was testing manually, but I will see if it works and also check `fstab` I guess.

Reply:
No, `/run` is not RO